src/Accounts/Accounts/CommonModule/ContextAdapter.cs

```diff
@@ -118,7 +121,12 @@ internal void AddAuthorizeRequestHandler(
                 {
                     endpointResourceIdKey = endpointResourceIdKey ?? AzureEnvironment.Endpoint.ActiveDirectoryServiceEndpointResourceId;
                     var context = GetDefaultContext(_provider, invocationInfo);
-                    return await AuthenticationHelper(context, endpointResourceIdKey, endpointSuffixKey, request, cancelToken, cancelAction, signal, next);
+                    ICmdletContext cmdletContext = AzureCmdletContext.CmdletNone;
```

Comment:
AddAuthorizeRequestHandler is used in Autogen Data plane module. But extensibleParameters is actually not populated when this function is called in autogen module. So for autogen modules, we will not generate telemetry for authentication, is it right?

Reply:
As @dolauli  proposed, the auto-gen cmdlets are not derived from AzurePSCmdlet and so to enable authentication telemetry for auto-gen cmdlets, more changes should be done. I am going to submit a separated PR to implement it.